packages/expo-task-manager/src/ExpoTaskManager.web.ts

```diff
@@ -7,4 +7,7 @@ export default {
   },
   addListener() {},
   removeListeners() {},
+  async isAvailableAsync(): Promise<boolean> {
+    return true;
```

Comment:
```suggestion
    return false;
```
I don't think task manager is available on web. 😃